Sources/AirbnbSwiftFormatTool/AirbnbSwiftFormatTool.swift

```diff
@@ -110,7 +110,7 @@ struct AirbnbSwiftFormatTool: ParsableCommand {
     }
 
     if let swiftVersion = swiftVersion {
-      arguments += ["--swiftversion", swiftVersion]
```

Comment:
`--swiftversion` is the correct / preferred capitalization according to the SwiftFormat documentation: https://github.com/nicklockwood/SwiftFormat#swift-version

Reply:
Thank you for the review:)

I've reverted to the original state in response to the review, making only the necessary change to add the missing `r`.